core/ecschema-metadata/src/IncrementalLoading/SchemaParser.ts

```diff
@@ -34,14 +34,14 @@ export class SchemaParser {
    * @param context The SchemaContext that will contain the schema and it's references.
    * @returns The corrected SchemaProps JSON.
    */
-  public static async parse(schema: SchemaProps, context: SchemaContext): Promise<SchemaProps> {
+  public static async parse(schema: SchemaProps, schemaInfos: Iterable<SchemaInfo>): Promise<SchemaProps> {
```

Comment:
Is this a breaking change?

Reply:
The class is internal, so no breaking change.

Reply:
Ah missed the internal. Thanks.